unidist/core/backends/mpi/core/async_operations.py

```diff
@@ -5,15 +5,13 @@
 import unidist.core.backends.mpi.core.common as common
 import unidist.core.backends.mpi.core.communication as communication
 
+mpi_state = communication.MPIState.get_instance()
 # Logger configuration
 # When building documentation we do not have MPI initialized so
 # we use the condition to set "worker_0.log" in order to build it succesfully.
-log_file = "worker_{}.log".format(
-    communication.MPIState.get_instance().rank
-    if communication.MPIState.get_instance() is not None
-    else 0
-)
-logger = common.get_logger("worker", log_file)
+logger_name = "worker_{}".format(mpi_state.rank if mpi_state is not None else 0)
+log_file = "{}.log".format(logger_name)
+logger = common.get_logger(logger_name, log_file)
```

Comment:
```suggestion
logger = common.get_logger("async_operations", "async_operations.log")
```
Since mpi_state is always None here, we can just use the file name as is.